spotify/models/user.py

```diff
@@ -28,6 +30,63 @@
 REFRESH_TOKEN_URL = "https://accounts.spotify.com/api/token?grant_type=refresh_token&refresh_token={refresh_token}"
 
 
+class TokenInfo:
```

Comment:
Is `TokenInfo` intended to be a public mechanism? its use in a constructor `from_token_info` make it seem so but It's not exposed in the models `__init__` and its defined in this source file, where more commonly `modes/common.py` would be used.

Reply:
I think that having it public would unify the interaction with the token mechanism, 
and it allows to later on manually extract information about the tokens/expiry times from the `User` object. I'll move it to `models/common.py` and export it.